src/applications/vaos/tests/mocks/fetch.js

```diff
@@ -0,0 +1,130 @@
+/** @module testing/mocks/fetch */
```

Comment:
I was thinking that it made sense to use mocks/data for our mock data functions that work by version, and mock/fetch for fetch mock helpers that can work by version. And eventually, we'd move everything into those two files, since that naming convention I think makes more sense than our current one.